client/src/components/modals_and_popovers/StatelessModal.tsx

```diff
@@ -96,7 +96,6 @@ export class StatelessModal extends React.Component<StatelessModalProps> {
           {body && <Modal.Body>{body}</Modal.Body>}
           {footer_content && <Modal.Footer>{footer_content}</Modal.Footer>}
         </div>
-        <div tabIndex={0} onFocus={this.closeModal} />
       </Modal>
```

Comment:
I think this was important, so that "leaving" the modal (with any keyboard navigation) would close it and send focus back to wherever it had been before.

Reply:
Hmm, I'm not sure if tabbing to leave modal should be the correct behaviour, especially because user would've had to purposely tab-past "cancel" button.

Without this div, tab goes to address bar, then comes back into the modal. User has to either press esc or trigger cancel to leave modal. Focus goes back to where it should go regardless

But I'll just disable rule and leave the div if you prefer!